src/actions/FirstOpenActions.tsx

```diff
@@ -16,14 +16,21 @@ const asAppleAdsAttribution = asObject({
 })
 type AppleAdsAttribution = ReturnType<typeof asAppleAdsAttribution>
 
-const asFirstOpenInfo = asObject({
-  isFirstOpen: asValue('true', 'false'),
+interface FirstOpenInfo {
+  isFirstOpen: boolean
+  deviceId: string
+  firstOpenEpoch: number
+  countryCode?: string
+  appleAdsAttribution?: AppleAdsAttribution
+}
+type FirstOpenInfoFile = Omit<FirstOpenInfo, 'appleAdsAttribution'>
+
+const asFirstOpenInfoFile = asObject<FirstOpenInfoFile>({
+  isFirstOpen: asBoolean,
```

Comment:
Oh, shoot! I didn't see that were quoted strings on disk. Ok, we can still be backwards compatible using a cleaner like this:

```ts
const asStringBool: Cleaner<boolean> = (raw) => {
  if (typeof raw === 'string'){
    if (raw === 'true') return true
    if (raw === 'false') return false
  }
  return asBoolean(raw)
}
```

This translate the existing data into a boolean on the next open. Otherwise, we could go back to `asValue('true', 'false')`, which is insane but what we have.